src/views/Dashboard.tsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
+import DashboardHeader from '../components/DashboardHeader';
 
 const Dashboard: React.FunctionComponent = () => {
-    return <h1>Dashboard</h1>;
+    return (
+        <h1>
+            Dashboard
+            <DashboardHeader />
+        </h1>
```

Comment:
DashboardHeader shouldn't be wrapped inside the h1. React will complain that there are 2 non-nested components so you can wrap everything in React.Fragment

Reply:
Fixed.